src/helpers/formatWithTitleCase.ts

```diff
@@ -0,0 +1,13 @@
+export const formatWithTitleCase = (
+  input: string | undefined,
+): string | undefined => {
+  if (input === undefined) {
+    return undefined;
+  }
```

Comment:
It might be safer to return an empty string instead of `undefined` here.

```suggestion
  input: string | undefined,
): string => {
  if (!input) {
    return "";
  }
```

For example, if this is used in a string literal, it will output `undefined`:

```typescript
<div>{`Some text ${formatWithTitleCase(undefined)}`}</div> // Some text undefined
```

Otherwise, we'll need to do extra check where it is used:

```typescript
<div>{`Some text ${formatWithTitleCase(undefined) || ""}`}</div> // Some text
```

Reply:
Cool, TY!